be/src/agent/finish_task.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+namespace starrocks {
+
+class TFinishTaskRequest;
+
+void finish_task(const TFinishTaskRequest& finish_task_request);
```

Comment:
This function name is too common, do we need to give it a namespace or make it a class static function?

Reply:
The function parameter types are part of the function signature,  no naming conflict.